### ISSUE TITLE
Add shortcuts to improve the ergonomics of the android mapper application

### ISSUE THREAD

**Issue by eolmapper** [OpenOrienteering/mapper#2242]
I use the mapper androit app every day and there are operations that I repeat all the day. 
New shortcuts could streamline these operations and save my time, and I hope that of many other users!

The problem is that we have to go to the templates menu to access to "hide all templates" or "hide any template", and then close the template menu.  (3 actions)

-First suggest is to have a shortcut button "hide all template" in the toolbar next to "show baselines" (equivalent to the F10 shortcut in the desktop interface)
we need this to clearly see what we have drawn, because sometimes templates do not allow this.

-second suggest is to have favorites templates we can switch "show" or "hide" with button in toolbar.
This allow you to switch between templates in 1 action (e.g. ortophoto and slope shading)

In the template control menu, a check box "add to favorite"  should be added per line 
A button in the toolbar will  active "show/hide" for all of the templates that have been added as favorites.

I would like to help as much as possible to add these functionalities (at least the first suggestion seems quite simple to me) but I do not have the skills to code. I can prepare the icons, do beta testing or anything that might be useful.

**Comment by mlerjen** [comment]
relates to #534
I had proposed to have three definable presets of basemap-mixes to directly (one click) access from the main screen...

**Comment by eolmapper** [comment]
> relates to #534 I had proposed to have three definable presets of basemap-mixes to directly (one click) access from the main screen...

yes it's the same idea, however for the rest I find that the current interface suits me very well.

**Comment by dl3sdo** [comment]
For the desktop version, I imagine a GUI like this:

https://github.com/user-attachments/assets/34cd7f37-c633-4244-8d0b-3d5b73a00587
